sqlglot/expressions.py

```diff
@@ -1428,10 +1442,12 @@ class Properties(Expression):
     arg_types = {"expressions": True}
 
     NAME_TO_PROPERTY = {
```

Comment:
i'm not a big fan of the NAME_TO_PROPERTY dictionary. why isn't this just a property on on the property?

Reply:
Agreed, it doesn't feel very elegant. On the other hand, we used to store the property name in `this`, which wasn't very elegant either.